test/Microsoft.AspNet.Diagnostics.Tests/project.json

```diff
@@ -4,6 +4,9 @@
     },
     "dependencies": {
         "Microsoft.AspNet.Diagnostics.Elm": "1.0.0-*",
+        "Microsoft.AspNet.FileProviders.Embedded": "1.0.0-*",
+        "Microsoft.AspNet.StaticFiles": "1.0.0-*",
```

Comment:
Middleware repos should avoid taking dependencies on other middleware repos. It complicates the CI dependency chain. You should be able to mimic the behavior of StaticFiles accurately enough for testing.